docs/interchain-staking.md

```diff
@@ -0,0 +1,178 @@
+
+Limits on a child chain's ability to slash a parent chain validator may also be imposed at the parent chain level. For instance it is possible to make it a required parameter of a child chain within the Chain Registry to provide a maximum slashing rate. This would ensure that a validator knows even if they violate all the rules of a child chain, they can still limit the damage to their stake by some amount. It is also possible to enforce a parameter at the level of the parent chain that prevents a validator from joining any combination of child chain validator sets that results in a combined slashable rate over some threshold. For instance if there are 3 child chains that each have maximum slashable rates of 33% over X blocks, and the parent chain has a safety limit for validators that require them to stay below a total of 90% slashable events over X blocks, no validator would be allowed to join the Chain Registry for all 3 of those child chains. This parameter on the parent chain should be a governance parameter that can be adjusted by token staked voting to reflect the risk appetite of the actual validator set of the parent chain.
+
+Consideration should be made to the incentives around submitting evidence in order to ensure that punishable offenses do not go unnoticed. This is a similar issue to Relayer Incentivization, where currently it costs some fee to relay IBC packets but there is no reward or payment possible as part of the core IBC logic. As a result IBC packets are currently relayed in an altruistic manner. It's important to ensure that for integrity of the operation of the child and parent chain that slashable offenses are always submitted. It may be that the slashable amounts of tokens are used as rewards for submitting the evidence (assuming it's possible to ensure it is not the culprit submitting their own evidence and regaining their stake). Maybe some flat fee to at least pay for the transactions are required, although this may become redundant with the addition of any Relayer incentivizations into core IBC.
```

Comment:
Maybe this is already covered in relayer incentivization, but I guess there should also be some incentive for relayers to relay the messages that carry the validator set updates from the parent to the child chain, right?